src/providers/data_provider/dp_request.c

```diff
@@ -133,6 +133,10 @@ static errno_t dp_attach_req(struct dp_req *dp_req,
         SSS_REQ_TRACE_CID_DP_REQ(SSSDBG_TRACE_FUNC, dp_req->name,
                                  "New request. [%s CID #%u] Flags [%#.4x].",
                                  sender_name, cli_id, dp_flags);
+        if (be_is_offline(provider->be_ctx)) {
+            DEBUG(SSSDBG_TRACE_FUNC, "WARNING: [CID #%u] Backend is offline! " \
+                                     "Requests may fail\n", cli_id);
```

Comment:
Hi,

I think `WARNING:` and `Request may fail` are a bit to strong and might lead to wrong conclusions since being in offline mode is in general not an error.

I would suggest something like `Backend is offline! Using cached data if available`.

bye,
Sumit

Reply:
Fixed.